### PR TITLE
staking: update log level

### PR DESCRIPTION
**Pull request by anshalshukla**
# Description

Issue: [#1771](https://polygon.atlassian.net/browse/POS-1771?atlOrigin=eyJpIjoiMjg3YzdhNTZjMGU5NDUyM2JjYWM4NTBmZjE5OTk0NWMiLCJwIjoiaiJ9)

# Changes

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] Hotfix (change that solves an urgent issue, and requires immediate attention)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)
- [ ] Changes only for a subset of nodes

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it

# Nodes audience

In case this PR includes changes that must be applied only to a subset of nodes, please specify how you handled it (e.g. by adding a flag with a default value...)

# Checklist

- [x] I have added at least 2 reviewer or the whole pos-v1 team
- [x] I have added sufficient documentation in code
- [x] I will be resolving comments - if any - by pushing each fix in a separate commit and linking the commit hash in the comment reply

# Cross repository changes

- [ ] This PR requires changes to bor 
  - In case link the PR here:
- [ ] This PR requires changes to matic-cli
  - In case link the PR here:
- [x] This PR requires changes to Tendermint
  - In case link the PR here: [PR#27](https://github.com/maticnetwork/tendermint/pull/27)

## Testing

- [ ] I have added unit tests
- [ ] I have added tests to CI
- [ ] I have tested this code manually on local environment
- [ ] I have tested this code manually on remote devnet using express-cli
- [ ] I have tested this code manually on mumbai
- [ ] I have created new e2e tests into express-cli

### Manual tests

Please complete this section with the steps you performed if you ran manual tests for this functionality, otherwise delete it

# Additional comments

Update go.mod once the changes in Tendermint are merged

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -142,7 +142,7 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 )
 
-replace github.com/tendermint/tendermint => github.com/maticnetwork/tendermint v0.26.0-dev0.0.20230719144702-2a4b4a5a8b55
```
Comment (Raneet10) [inline go.mod:145]: Please switch back to the tendermint version we're using.

File: go.mod [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module github.com/maticnetwork/heimdall
 
-go 1.20
```
Comment (Raneet10) [inline go.mod:3]: Any reason why we are downgrading to `go 1.19 `?

File: go.mod (hunk 1 above)
Comment (Raneet10) [reply go.mod:145]: Oh I see you'll be updating it anyway, nvm.

File: go.mod [hunk 3]
```diff
@@ -142,7 +142,7 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 )
 
-replace github.com/tendermint/tendermint => github.com/maticnetwork/tendermint v0.26.0-dev0.0.20230719144702-2a4b4a5a8b55
+replace github.com/tendermint/tendermint => /Users/ashukla/polygon/tendermint
```
Comment (marcello33) [inline go.mod:145]: Can this go under `maticnetwork` org?

File: go.mod (hunk 2 above)
Comment (anshalshukla) [reply go.mod]: I had just updated it to perform go mod tidy as locally I had go 1.19, have reverted it back

**Review by Raneet10 — APPROVED** [review]

**Review by manav2401 — APPROVED** [review]

**Pull request merged**